mitodl/mit-open
# Data fixtures app for loading static fixtures

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4687

### Description (What does it do?)
This PR adds a "data fixtures" app that exists for the purpose of updating/loading static fixtures in both local and prod environments. for discussion around this reference the discussion topic https://github.com/mitodl/hq/discussions/4747

The idea is that this app will run its data migrations for local development as well as when deploying but not when running tests

In addition to adding the app, this PR also add initial fixture data for testimonials (copied from production)

### How can this be tested?
1. Checkout this branch
2. do a docker compose down/up and note that fixtures for attestation are loaded in via django admin. alernatively if you want to avoid doing a docker down/up you can run ```RUN_DATA_MIGRATIONS=true python manage.py migrate```
